lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -62,6 +62,16 @@ function gutenberg_generate_block_templates_export_file() {
 		wp_json_encode( $tree->get_data(), JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE )
 	);
 
+	$zip->addFromString(
+		'theme/index.php',
+		'<?php // This file is needed to make the theme work, but it doesn\'t do anything.'
```

Comment:
Can we go with "Silence is golden"?

Reply:
I wonder if the best thing to do is the same as for the style.css - just copy the index.php